requirements/requirements-test.txt

```diff
@@ -19,4 +19,5 @@ datasets
 pydantic
 ray
 peft>=0.7.1
+rpyc==6.0.0
```

Comment:
As this is in `requirements.txt`, it's redundant here.

Reply:
Fixed in #5744